CHANGELOG.md

```diff
@@ -1,4 +1,46 @@
-## 3.8.0 (December 31, 2020)
+## 3.8.1 (April 11, 2025)
+
+BUG FIXES:
```

Comment:
Missing: 
PR 1585 - Fix upgrade pause issue, where upgrade process doesn't pause in the end of a component processing.